docs/src/modules/components/MaterialFreeTemplatesCollection.js

```diff
@@ -100,7 +100,8 @@ export default function Templates() {
                 borderColor: 'divider',
               }}
             >
-              <Box
+              <Link
+                href={layout.href}
```

Comment:
Make the whole image clickable, reduce the distance the mouse has to travel, so much more enjoyable on my end 😌.